Libplanet.Net/Transports/NetMQTransport.cs

```diff
@@ -832,7 +837,10 @@ private async Task ProcessRequest(MessageRequest req, CancellationToken cancella
                     await Task.Delay(1000);
                 }
 
-                Interlocked.Decrement(ref _socketCount);
+                if (dealer is { IsDisposed: true })
+                {
+                    Interlocked.Decrement(ref _socketCount);
+                }
```

Comment:
I'm not sure disposal check is any better than a simple `null` check. For whatever reason if sockets are not disposed properly, we might overcount the number of `DealerSocket`s instead. I think `ITransport<T>` layer should only be concerned with its own references if possible, and try not to deal with the underlying `NetMQ` layer. 😶

Reply:
At least, `NetMQSocket.Close()` seems to turn off `.IsDisposed` immediately. but... yeah I also think that a simple null check is enough in this context.